README.md

```diff
@@ -35,7 +35,7 @@ internals shift as we find better/more performant abstractions.
 
 We will guarantee compatibility between minor versions (i.e 0.0.1 => 0.0.2) but not major versions (0.0.1 => 0.1.0)
 
-0.0.1-M6 is here for scala 2.12+ and Cats 1.0.0-RC1!
+0.0.1-M7 is here for scala 2.12+ and Cats 1.0.0-RC2!
```

Comment:
Are we releasing immediately after the upgrade? Otherwise this is misleading.

Reply:
let me merge this one. I can make sure we release after update iff I merge this

Reply:
we are waiting for http4s :+1: